### DISCUSSION TITLE
Mistake that BT-630 Deadline for receipt of expressions of interest shall be mandatory for form 8?

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#199] [Q&A] [answered]
Hi,
According to the latest fields.json, BT-630 Deadline for receipt of expressions of interest shall be mandatory for form 8, 10, 11, 12, 13, 14, CEI. Should form 8 PIN time limit really be part of that rule? Isn't BT-630 only relevant to PIN CFC forms?

Kind regards, 
Åsa

**Comment by YvesJo** [comment, marked as answer]
Indeed, BT-630 shall not exist for Form Subtype 8. this will get fixed in a later release.
